### PR TITLE
6 -  /api/articles/:article_id/comments

### PR DESCRIPTION
**Pull request by drewfullstack**
adds GET comments by article_id route handler
adds relevant tests and error handling
updates endpoints.json

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -98,3 +98,42 @@ describe("GET /api/articles", () => {
...
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
+        });
+      });
+  });
+
+  test("status: 404 - responds with 404 not found for an article id that does not exist / article with no comments", () => {
```
Comment (nc-beth-mccormack) [inline __tests__/app.test.js:123]: Nice 404 for article id that does not exist✅
Looking at your test description, should an article with no comments be an error? Has something actually gone wrong with the request, or was it successful and the article doesn't have any comments to get?

File: models/articles.models.js [hunk 2]
```diff
@@ -19,3 +19,17 @@ exports.fetchArticles = () => {
       return rows;
     });
 };
+
+exports.fetchComments = (article_id) => {
```
Comment (nc-beth-mccormack) [inline models/articles.models.js:23]: This looks good well done!

**Review by nc-beth-mccormack — COMMENTED** [review]
I've left some comments 🙂

File: __tests__/app.test.js (hunk 1 above)
Comment (drewfullstack) [reply __tests__/app.test.js:123]: That makes sense! I have altered it to now return a 200 for an article with no comments :)

**Pull request merged**